extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -713,7 +713,6 @@ export class ModelEditorView extends AbstractWebview<
       this.cliServer,
       this.databaseItem.language,
       makeSelected,
-      false,
     );
     if (!addedDatabase) {
```

Comment:
I assume we no longer need to treat the model editor as a special case, so I've removed the argument here.

Reply:
I'm not sure about this change: if you are in the model editor and have the setting enabled, we would now suddenly reload when you try to model a dependency. I think this may be unexpected because the user might not know that the database gets added as a normal database.

Reply:
You're right that this might be unexpected. Since it's less explicit that you're adding a database in this case, I'll keep the hard-coded "don't add source archive" setting! 👍🏽